lib/src/udp/udp_main.c

```diff
@@ -246,6 +246,12 @@ static int udp_build_tx_msg_pkt(struct mtl_main_impl* impl, struct mudp_impl* s,
   int idx = s->idx;
   int ret;
 
+  // Ensure pkts_nb is greater than 0
+  if (pkts_nb == 0) {
```

Comment:
Why it need this check? If there's a real case where we can get zero pkts, the check should happen on upper `mudp_sendmsg` function.

And the comment style should be `/* xxx */`